File: launchdarkly/typescript-axios/src/GetMauUsageExample.ts
import axios, { AxiosError } from "axios";
import * as api from "launchdarkly_client"

const configuration = new api.Configuration({
  apiKey: "YOUR_API_KEY",
});

new api.AccountUsageBetaApi(configuration).getMauUsage().then(response => {
  console.log(response.data);
}).catch((error: Error | AxiosError) => {
  console.log("Exception when calling AccountUsageBetaApi#getMauUsage:");
  axios.isAxiosError(error) ? console.log(error.message) : console.log(error);
});

File: launchdarkly/java/src/main/java/oseg/launchdarkly_examples/GetMauUsageExample.java
package oseg.launchdarkly_examples;

import com.launchdarkly.client.ApiException;
import com.launchdarkly.client.Configuration;
import com.launchdarkly.client.api.*;
import com.launchdarkly.client.auth.*;
import com.launchdarkly.client.JSON;
import com.launchdarkly.client.model.*;

import java.io.File;
import java.math.BigDecimal;
import java.time.LocalDate;
import java.time.OffsetDateTime;
import java.util.ArrayList;
import java.util.List;
import java.util.Map;

public class GetMauUsageExample
{
    public static void main(String[] args)
    {
        var config = Configuration.getDefaultApiClient();
        ((ApiKeyAuth) config.getAuthentication("ApiKey")).setApiKey("YOUR_API_KEY");

        try
        {
            var response = new AccountUsageBetaApi(config).getMauUsage(
                null, // from
                null, // to
                null, // project
                null, // environment
                null, // sdktype
                null, // sdk
                null, // anonymous
                null, // groupby
                null, // aggregationType
                null // contextKind
            );

            System.out.println(response);
        } catch (ApiException e) {
            System.err.println("Exception when calling AccountUsageBetaApi#getMauUsage");
            System.err.println("Status code: " + e.getCode());
            System.err.println("Reason: " + e.getResponseBody());
            System.err.println("Response headers: " + e.getResponseHeaders());
            e.printStackTrace();
        }
    }
}
